C/plugins/storage/postgres/init.sql

```diff
@@ -773,99 +727,96 @@ INSERT INTO foglamp.log_codes ( code, description )
+            );
 
+-- HTTP South template
 INSERT INTO foglamp.configuration ( key, description, value )
-    VALUES ( 'CC2650POLL', 'SensorTagCC2650 South Poll Plugin Configuration', ' { "plugin" : { "type" : "string", "value" : "cc2650poll", "default" : "cc2650poll", "description" : "Python module name of the plugin to load" } } ');
+    VALUES ( 'HTTP_SOUTH',
+             'HTTP Listener South Plugin',
+             ' { "plugin" : { "type" : "string", "value" : "http_south", "default" : "http_south", "description" : "Python module name of the plugin to load" } } '
+           );
 
+-- COAP: CoAP Listener Plugin
 INSERT INTO foglamp.configuration ( key, description, value )
-    VALUES ( 'CC2650ASYN', 'SensorTagCC2650 South Async Plugin Configuration', ' { "plugin" : { "type" : "string", "value" : "cc2650async", "default" : "cc2650async", "description" : "Python module name of the plugin to load" } } ');
+     VALUES ( 'COAP',
+              'CoAP Listener South Plugin',
+              ' { "plugin" : { "type" : "string", "value" : "coap_listen", "default" : "coap_listen", "description" : "Python module name of the plugin to load" } } '
+            );
 
 INSERT INTO foglamp.configuration ( key, description, value )
-    VALUES ( 'HTTP_SOUTH', 'HTTP South Plugin Configuration', ' { "plugin" : { "type" : "string", "value" : "http_south", "default" : "http_south", "description" : "Python module name of the plugin to load" } } ');
-
--- DELETE data for roles, resources and permissions
-DELETE FROM foglamp.role_resource_permission;
-DELETE FROM foglamp.roles;
-DELETE FROM foglamp.resources;
+    VALUES ( 'CC2650POLL',
+             'TI SensorTag CC2650 polling South Plugin',
+             ' { "plugin" : { "type" : "string", "value" : "cc2650poll", "default" : "cc2650poll", "description" : "Python module name of the plugin to load" } } '
+           );
 
--- Roles
-INSERT INTO foglamp.roles ( id, name, description )
-     VALUES ( 1, 'Power User', 'A user with special privileges' );
+INSERT INTO foglamp.configuration ( key, description, value )
+    VALUES ( 'CC2650ASYN',
+             'TI SensorTag CC2650 async South Plugin',
+             ' { "plugin" : { "type" : "string", "value" : "cc2650async", "default" : "cc2650async", "description" : "Python module name of the plugin to load" } } '
+           );
 
--- Resources
-INSERT INTO foglamp.resources ( id, code, description )
-     VALUES ( 1, 'PURGE_MGR', 'Can Start / Stop the purging process' );
-INSERT INTO foglamp.resources ( id, code, description )
-     VALUES ( 2, 'PURGE_RULE', 'Can view or set purging rules' );
 
+-- STRMN: Streaming
+--        status      : the process is on or off, it is on by default
+--        time window : the time window when the process is active, always active by default (it means every second)
+INSERT INTO foglamp.configuration ( key, description, value )
```

Comment:
I don't believe this is used, since the sending process schedule takes care of this.